systemtests/pom.xml

```diff
@@ -168,7 +168,7 @@
                     <groups>${groups}</groups>
                     <excludedGroups>${excludeGroups}</excludedGroups>
                     <skipTests>${skip.tests}</skipTests>
-                    <forkCount>1</forkCount>
+                    <forkCount>0</forkCount>
```

Comment:
:(

Reply:
@kornys That was not intentional, just for local debugging. I will revert it :)